packages/components/src/core/styles/common/defaultTheme.ts

```diff
@@ -239,44 +252,193 @@ export const defaultAppTheme: AppTheme = {
+    "400": `1px solid ${lightAppTheme.colors.gray[400]}`,
+    "500": `1px solid ${lightAppTheme.colors.gray[500]}`,
+    dashed: `2px dashed ${lightAppTheme.colors.gray[400]}`,
+  },
+  link: {
+    dashed: `1px dashed`,
+    solid: `1px solid`,
+  },
+  primary: {
+    "300": `1px solid ${lightAppTheme.colors.primary[300]}`,
+    "400": `1px solid ${lightAppTheme.colors.primary[400]}`,
+    "500": `1px solid ${lightAppTheme.colors.primary[500]}`,
+    "600": `1px solid${lightAppTheme.colors.primary[600]}`,
+    dashed: `2px dashed ${lightAppTheme.colors.primary[400]}`,
+  },
+  success: {
+    "400": `1px solid ${lightAppTheme.colors.success[400]}`,
+  },
+  warning: {
+    "400": `1px solid ${lightAppTheme.colors.warning[400]}`,
+  },
+};
+
+// TODO: Dark mode colors
+darkAppTheme.borders = {
   error: {
-    "400": `1px solid ${defaultAppTheme.colors.error[400]}`,
+    "400": `1px solid ${lightAppTheme.colors.error[400]}`,
   },
   gray: {
-    "100": `1px solid ${defaultAppTheme.colors.gray[100]}`,
-    "200": `1px solid ${defaultAppTheme.colors.gray[200]}`,
-    "300": `1px solid ${defaultAppTheme.colors.gray[300]}`,
-    "400": `1px solid ${defaultAppTheme.colors.gray[400]}`,
-    "500": `1px solid ${defaultAppTheme.colors.gray[500]}`,
-    dashed: `2px dashed ${defaultAppTheme.colors.gray[400]}`,
+    "100": `1px solid ${lightAppTheme.colors.gray[100]}`,
+    "200": `1px solid ${lightAppTheme.colors.gray[200]}`,
+    "300": `1px solid ${lightAppTheme.colors.gray[300]}`,
+    "400": `1px solid ${lightAppTheme.colors.gray[400]}`,
+    "500": `1px solid ${lightAppTheme.colors.gray[500]}`,
+    dashed: `2px dashed ${lightAppTheme.colors.gray[400]}`,
   },
   link: {
     dashed: `1px dashed`,
     solid: `1px solid`,
   },
   primary: {
-    "300": `1px solid ${defaultAppTheme.colors.primary[300]}`,
-    "400": `1px solid ${defaultAppTheme.colors.primary[400]}`,
-    "500": `1px solid ${defaultAppTheme.colors.primary[500]}`,
-    "600": `1px solid${defaultAppTheme.colors.primary[600]}`,
-    dashed: `2px dashed ${defaultAppTheme.colors.primary[400]}`,
+    "300": `1px solid ${lightAppTheme.colors.primary[300]}`,
+    "400": `1px solid ${lightAppTheme.colors.primary[400]}`,
+    "500": `1px solid ${lightAppTheme.colors.primary[500]}`,
+    "600": `1px solid${lightAppTheme.colors.primary[600]}`,
+    dashed: `2px dashed ${lightAppTheme.colors.primary[400]}`,
   },
   success: {
-    "400": `1px solid ${defaultAppTheme.colors.success[400]}`,
+    "400": `1px solid ${lightAppTheme.colors.success[400]}`,
   },
   warning: {
-    "400": `1px solid ${defaultAppTheme.colors.warning[400]}`,
+    "400": `1px solid ${lightAppTheme.colors.warning[400]}`,
   },
 };
 
-export function makeThemeOptions(appTheme: AppTheme): SDSThemeOptions {
+const lightThemePalette = (appTheme: AppTheme): PaletteOptions => ({
+  divider: appTheme.colors.gray[200],
+  background: {
+    default: common.white,
+    secondary: appTheme.colors.gray[100],
```

Comment:
I think this was added by @hthomas-czi during the hackathon, it's throwing a type error because we don't have it on `TypeBackground`.

@hthomas-czi Is a secondary background still something that's needed? There's `paper` that we could be using that already exists, so we'll have three background colors if we do use `secondary`.

If it's needed, it's easy to add to the type. I just want to check 😁

Reply:
> I think this was added by @hthomas-czi during the hackathon, it's throwing a type error because we don't have it on `TypeBackground`.
> 
> @hthomas-czi Is a secondary background still something that's needed? There's `paper` that we could be using that already exists, so we'll have three background colors if we do use `secondary`.
> 
> If it's needed, it's easy to add to the type. I just want to check 😁

Hey @lanesawyer! Sorry for the delay responding. For some horrible reason, I don't receive notifications for GitHub mentions. 

I'll leave it up to you, but `Paper` doesn't fully address the reason I added a secondary background. They're equivalent in dark mode. But in light mode, `Paper` is always white, while `background[secondary]` is light gray in light mode. This maintains a consistent appearance between modes. 

Thanks!